test/jdk/javax/swing/JRadioButton/4380543/bug4380543.java

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2002,2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
guess there should be a space between 2002, and 2022 else validate-source task might fail.